lib/foreman_rescue/engine.rb

```diff
@@ -51,7 +65,7 @@ class Engine < ::Rails::Engine
     end
 
     initializer 'foreman_rescue.register_gettext', after: :load_config_initializers do |_app|
-      locale_dir = File.join(File.expand_path('../../..', __FILE__), 'locale')
+      locale_dir = File.join(File.expand_path('../..', __dir__), 'locale')
```

Comment:
Isn't this essentially the same?
```suggestion
      locale_dir = File.expand_path('../../locale', __dir__)
```

Reply:
Probably is. this was autocorrected by rubocop due to https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/ExpandPathArguments